app/models/task.py

```diff
@@ -1,5 +1,34 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from sqlalchemy import ForeignKey
 from ..db import db
+from datetime import datetime
+from typing import Optional
+
 
 class Task(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    title: Mapped[str]
```

Comment:
Same comment as above in `goal.py`, consider what columns should be allowed to be nullable and which ones should not be nullable. Make sure to set the appropriate constraints on the attributes. I'd argue that a client cannot create a task without a title so you'd want line 10 to look like:
```suggestion
    title: Mapped[str] = mapped_column(String, nullable=False)
```